uint8 color
float32 probability
uint8 CONE_COLOR_BLUE = 0
uint8 CONE_COLOR_YELLOW = 1
uint8 CONE_COLOR_RED = 2
uint8 CONE_COLOR_SMALL_ORANGE = 3
uint8 CONE_COLOR_BIG_ORGANGE = 4
uint8 CONE_COLOR_UNKNOWN = 5
#Comment: If you change the CONE_COLOR enum make sure to also change the code in see_camera_processing/coordinate_extraction_node